packages/storybook/stories/open-forms/StoryUtil.tsx

```diff
@@ -62,3 +66,7 @@ export const SingleFormioComponent = ({
   const component = merge({ type, key, label }, extraComponentProperties);
   return <RenderFormioForm form={{ type: 'form', components: [component] }} />;
 };
+
+export const MultipleFormioComponents = ({ components }: PropsWithChildren<MultipleFormioComponentsProps>) => {
```

Comment:
`PropsWithChildren<>` is hier niet nodig, we gebruiken geen children prop. Alleen `MultipleFormioComponentsProps` als type is voldoende. (Hetzelfde geldt voor de `SingleFormioComponent`